libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -35,17 +36,336 @@ const selector = 'eo-landing-page-shell';
+            padding-bottom: calc(2 * var(--watt-space-xl));
+          }
+        }
+        .${selector}__content--centered {
+          text-align: center;
+        }
+        .${selector}__one-column {
+          width: calc(100% - 200px);
+          margin: 0 auto;
+        }
+        .${selector}__column--large {
+          width: 60%;
+          display: inline-block;
+        }
+        .${selector}__column--small {
+          width: 40%;
+          display: inline-block;
+        }
+
+        .${selector}__link {
+          display: inline-block;
+          color: var(
+            --watt-color-primary
+          ); // This overrides the '--watt-color-primary-dark' color which is currently added by the watt-text-s class
+        }
+
+        // 1. Primary Watt Button.
+        // 2. Normal size Watt Button.
+        // 3. Custom size for Watt Button in App bar.
+        // 4. Align text vertically.
+        .${selector}__login-link {
+          @include watt.typography-watt-button; // [1]
+
+          --height: calc(10 * var(--watt-space-xs));
+          --inset-squish-m--x: var(--watt-space-m);
+          --inset-squish-m--y: var(--watt-space-s);
+
+          background: var(--watt-color-primary); // [1]
+          color: var(--watt-color-primary-contrast); // [1]
+
+          min-width: 6.25rem; // [2]
+          height: var(--height); // [3]
+          padding: var(--inset-squish-m--y) var(--inset-squish-m--x); // [3]
+
+          line-height: calc(
+            var(--height) - 2 * var(--inset-squish-m--y)
+          ); // [3] [4]
+
+          &:hover {
+            text-decoration: none; // [1]
+          }
+        }
+
+        .font-primary-color {
+          color: var(--watt-color-primary);
+        }
+
+        .${selector}__p {
+          @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
+        }
+
+        .${selector}__h1 {
+          @include watt.typography-watt-headline-1; // This overrides the styles applied from Angular Material on h1 tags
+        }
+
+        .${selector}__h2 {
+          @include watt.typography-watt-headline-2; // This overrides the styles applied from Angular Material on h2 tags
+          text-transform: none; // This overrides the uppercased styling from watt
+        }
+
+        .${selector}__img {
+          width: 100%;
+          height: auto;
+        }
+
+        .${selector}__footer-mesh {
+          width: 35%;
+          position: absolute;
+          bottom: 200px; // Position the image just above the footer
+        }
+
+        .${selector}__header-mesh {
+          width: 35%;
+          position: absolute;
+          top: 64px; // Position the image just below the header
+          right: 0;
+        }
+
+        .${selector}__development-notification {
+          background: rgba(223, 235, 218, 0.9);
+          top: -40px;
+          width: 100%;
+          height: 80px;
```

Comment:
suggestion: use factors of the base Watt spacing unit (X * 4px)

Reply:
Updated